backend/src/controller/chatController.ts

```diff
@@ -283,15 +282,18 @@ async function handleChatToGPT(req: OpenAiChatRequest, res: Response) {
 	}
 
 	let updatedChatHistory = levelResult.chatHistory;
-	totalSentEmails.push(...levelResult.chatResponse.sentEmails);
+
+	const totalSentEmails: EmailInfo[] = [
```

Comment:
before, this was being instantiated then mutated once to add on the new emails sent as a result of the message. Now we just instantiate it including the new emails